gitbook-docs/market/overview.md

```diff
@@ -1,63 +1,81 @@
-  * **Circuit**: The instruments for which this order is placed. ex: MINA state-proof circuit or Solana state-proof circuit.
-  * **Cost**: Amount the user is willing to pay for the proof. This is a number represented in USD.
-  * **Order Time Out**: Once the order enters the marketplace, this defines how long it should stay in the order book. If it times out before being matched, it is purged. This is a number expressed in minutes.
-  *   **Generation Time**:  Once the order is matched, this specifies the maximum time the proof generator has to publish the proof. This is an optional parameter. This is a number expressed in minutes.&#x20;
+## Orders types
 
+* **Buy order**: A buy order is posted by a proof requester.
+    Each buy order comprises the following parameters:
+  * **Statement** — the instrument for which this order is placed, like MINA or Solana state-proof
+    circuit, with additional parameters like name, URL, etc.
+  * **Cost** — the amount in USD that the user is willing to pay for the proof.
+  * **Order time out** — time in minutes; how long should the order stay in the order book
+      once it enters the marketplace.
+    If it times out before being matched, the order will be purged.
+  * **Generation time** — the maximum time the proof generator has to publish the proof once the order
+      is matched.
+      This is an optional parameter expressed in minutes.
 
-* **Sell Order**: A sell order is posted by a proof Generator. Each sell order comprises of following parameters.
-  * **Circuit**: The instruments for which a generator is willing to generate a proof.
-  * **Cost:** Amount the proof generator is willing to accept to create a proof.
-  * **Order Time Out**: Once the order enters the marketplace, this defines how long it should stay in the order book. If it times out before being matched, it is purged. This is a number expressed in minutes.
-  * **Generation Time:** The upper bound by which the proof will be generated once matched.
 
-## Order Matching
+* **Sell order**: A sell order is posted by a proof generator.
+    Each sell order has the following parameters:
+  * **Statement** — the instruments for which a generator is willing to generate proof.
+  * **Cost** — the amount in USD that the proof generator is willing to accept to create a proof.
+  * **Order time out** — time in minutes; how long should the order stay in the order book
+    once it enters the marketplace.
+    If it times out before being matched, the order will be purged.
+  * **Generation time** — the maximum time the proof generator has to publish the proof once the order
+    is matched.
 
-Currently, a daemon process matches the orders when the following parameters are met within the order timeout criteria.
+## Order matching
 
-* Cost
-* Generation time
+Currently, a daemon process matches the orders when the following parameters are met
```

Comment:
This looks like an abstraction leaking. Do we need to know that there's a daemon process behind the Proof Market? There is a server application that might work as a daemon. If it changes to a container working under Kubernetes, will the order-matching rules change?

Reply:
This can be re-named to matching engine. Exchanges utilize different algos/processes to match/clear orders to what our process is doing hence the mention, but perhaps as you mention un-necessary.